specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5280,6 +5436,75 @@
       },
       "description": "A skill that can call a Web API endpoint, allowing you to extend a skillset by having it call your custom code."
     },
+    "CustomEntityLookupSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Text.CustomEntityLookupSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Skill"
+        }
+      ],
+      "properties": {
+        "entitiesDefinitionUri": {
+          "type": "string",
+          "description": "The uri pointing to a hosted json or csv file containing which entities to lookup."
```

Comment:
>hosted [](start = 48, length = 6)

nit: We can probably get rid of "hosted" and the description would still retain meaning